### PR TITLE
profiles: unmask virtual/jdk:17 and virtual/jre:17

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: profiles/base/package.use.stable.mask [hunk 1]
```diff
@@ -4,6 +4,12 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-05-08)
```
Comment (Flowdalic) [inline profiles/base/package.use.stable.mask:7]: I actually wonder if this mask is really necessary. Ideally we would start a tinder run with system-vm jdk17 and see which packages incorrectly specify a (build) dependency on jdk17.

Comment (vaukai) [reply profiles/base/package.use.stable.mask:7]: Isn't that mask needed until `virtual/jdk:17` and `virtual/jre:17` are stable on all supported arches?

File: profiles/base/package.use.stable.mask [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+# Having a masked gentoo-vm flag allows us to unmask virtual/{jdk,jre}:17
+# without breaking Gentoo packages.
```
Comment (gyakovlev) [inline profiles/base/package.use.stable.mask:9]: this is user-visible message, not a code comment.
please change to something like
`gentoo-vm useflag is not stable yet`

File: profiles/base/package.use.stable.mask (hunk 1 above)
Comment (gyakovlev) [reply profiles/base/package.use.stable.mask:7]: it's nice to have for safer transition, when virtual starts getting stable keywords.

File: profiles/base/package.use.stable.mask (hunk 2 above)
Comment (vaukai) [reply profiles/base/package.use.stable.mask]: Done.

**Pull request closed without merging**